core/packages/contracts/contracts/BasicInboundChannel.sol

```diff
@@ -5,61 +5,45 @@ import "./ParachainClient.sol";
 import "./utils/MerkleProof.sol";
 
 contract BasicInboundChannel {
-    uint256 public constant MAX_GAS_PER_MESSAGE = 100000;
-    uint256 public constant GAS_BUFFER = 60000;
-
     mapping(bytes32 => uint64) public nonce;
 
     ParachainClient public parachainClient;
 
-    struct MessageBundle {
-        bytes32 account;
-        uint64 nonce;
-        Message[] messages;
-    }
-
     struct Message {
-        uint64 id;
-        address target;
+        bytes32 sourceId;
```

Comment:
We use this style in Solidity: https://docs.soliditylang.org/en/v0.8.17/style-guide.html#naming-styles

```suggestion
        bytes32 sourceID;
```

Reply:
Got it. Updated the tests as well for consistency, even though they're in Typescript.